plenum/cli/cli.py

```diff
@@ -511,7 +511,7 @@ def _addOldGenesisCommand(self, matchedVars):
         txn = {
             TXN_TYPE: typ,
             TARGET_NYM: destId,
-            TXN_ID: sha256(randomString(6).encode()).hexdigest(),
+            # TXN_ID: sha256(randomString(6).encode()).hexdigest(),
```

Comment:
Why we commented this line? If we really don't need it, then it's better to put a comment describing why TXN_ID is not required there.